packages/melos/test/commands/format_test.dart

```diff
@@ -46,6 +48,7 @@ void main() {
 
       logger = TestLogger();
       final config = await MelosWorkspaceConfig.fromWorkspaceRoot(workspaceDir);
+      expectedExitCode = currentPlatform.isWindows ? 0 : 1;
```

Comment:
Why is the expected exit code 0 for windows? :thinking:

Reply:
Exactly, that's what I'm trying to figure out. I'm not really sure what's going on. It's the same issue in the `run_test` and even the `windows_test` step in the pipeline have this problem, exit code it's 1 but is 0 in windows environments.